packages/forms-web-app/src/views/projects/representations.njk

```diff
@@ -40,7 +40,7 @@
               </p>
               <div class="govuk-grid-row document-metadata">
                 <div class="govuk-grid-column-one-third" data-cy="published-date">
-                  {{ representation.DateRrepReceived | date }}
```

Comment:
Just a suggestion but you could leave this `date` filter in place, and just change the format which it is using. That seems to be defined in `packages/forms-web-app/src/config.js` on line 6.